src/main/scala/org/globalforestwatch/layers/Bgc2000.scala

```diff
@@ -3,6 +3,10 @@ package org.globalforestwatch.layers
 import org.globalforestwatch.grids.GridTile
 
 case class Bgc2000(gridTile: GridTile, model: String = "standard") extends FloatLayer with OptionalFLayer {
-  val model_suffix = if (model == "standard") "" else s"__$model"
-  val uri: String = s"$basePath/gfw_belowground_carbon_stock_2000${model_suffix}/v20191106/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/Mg/geotiff/${gridTile.tileId}.tif"
+  //      val model_suffix = if (model == "standard") "" else s"__$model"
+    //      val model_suffix = if (model == "standard") "" else s"__$model"
+  val model_suffix: String = if (model == "standard") "standard" else s"$model"
```

Comment:
Codacy found an issue: [Field name 'model_suffix' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6075981)